### ISSUE TITLE
Error isn't reported when **kwargs is wrong type

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#1861]
```python
def func(
    y: str,
    z: bool = ...,
) -> None:
    ...

kwargs: Dict[str, int] = {}
y1 = func(z=False, **kwargs)
```

If a `**kwargs` argument does not match the remaining required keyword arguments, an error is not reported.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.139, which I've just published. It will also be included in the next release of pylance.
